### ISSUE TITLE
Back to product categories disappears 

### ISSUE THREAD

**Issue by Delphinworks** [woocommerce/woocommerce-admin#3964]
**Describe the bug**
After working within a product category the "back to categories" button near top disappears.

**To Reproduce**
1. Go to 'product categories'
2. Click on 'customize category'
3. SAVE
4. Try to navigate back with <-- back to categories  ---- disappears

**Screenshots**

**Expected behavior**

**Isolating the problem (mark completed items with an [x]):**
- [ ] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [ ] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [ ] I can reproduce this bug consistently using the steps above.

**WordPress Environment**
<details>
```
Copy and paste the system status report from **WooCommerce > System Status** in WordPress admin.
```
</details>

**Comment by mattsherman** [comment]
#4019 should fix this

**Comment by mattsherman** [comment]
Fixed by #4019
